src/App.js

```diff
@@ -0,0 +1,160 @@
+import React, { Component } from "react";
+import {
+  Container,
+  Row,
+  Col,
+  Form,
+  Label,
+  Input,
+  InputGroup,
+  InputGroupAddon,
+  Button,
+} from "reactstrap";
+import roundTo from "round-to";
+import "./App.css";
+
+class App extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      advancedVersion: false,
+      billAmount: "",
+      numberOfPeople: "",
+      firstPayPercent: "",
+      firstPersonAmount: 0,
+    };
+    this.toggleAdvanced = this.toggleAdvanced.bind(this);
```

Comment:
Instead of using bind, in ES6 you can use arrow functions and there is no need to bind. For example, if you used toggleAdvanced(()=>{}), you would not have to bind

Reply:
agree